sdk/dotnet/Pulumi.Automation/PublicAPI.Shipped.txt

```diff
@@ -351,6 +351,7 @@ abstract Pulumi.Automation.Workspace.EnvironmentVariables.set -> void
 abstract Pulumi.Automation.Workspace.GetAllConfigAsync(string stackName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<System.Collections.Immutable.ImmutableDictionary<string, Pulumi.Automation.ConfigValue>>
 abstract Pulumi.Automation.Workspace.GetConfigAsync(string stackName, string key, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<Pulumi.Automation.ConfigValue>
 abstract Pulumi.Automation.Workspace.GetProjectSettingsAsync(System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<Pulumi.Automation.ProjectSettings>
+abstract Pulumi.Automation.Workspace.GetStackOutputsAsync(string stackName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.Task<System.Collections.Immutable.ImmutableDictionary<string, Pulumi.Automation.OutputValue>>
```

Comment:
Is this an unrelated change? Should it be in this PR?

Reply:
I can remove this if required - it's tidying up previously shipped apis that weren't moved from unshipped to shipped.

Reply:
Here or not, we should indeed move them.

Reply:
Understood! Sorry for confusion here.